### PR TITLE
Add idleTimeout, role and maxFrameSize settings for connectors

### PR DESCRIPTION
**Pull request by lulf**
Expose a few useful settings to address space connectors.

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated

File: address-space-controller/src/main/java/io/enmasse/controller/router/config/Connector.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
     private String saslMechanisms;
     private String failoverUrls;
     private Role role;
+    private Integer maxFrameSize;
```
Comment (k-wall) [inline address-space-controller/src/main/java/io/enmasse/controller/router/config/Connector.java:23]: Do you intend to make the corresponding change to the Go types too?

Comment (lulf) [reply address-space-controller/src/main/java/io/enmasse/controller/router/config/Connector.java:23]: Thanks for reminding me!

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated

**Pull request merged**